mentors/cpp/a.json

```diff
@@ -14,5 +14,14 @@
         "link_url": null,
         "avatar_url": null,
         "bio": "Until now I have worked mainly with C. However, through my master in Embedded Systems I discovered the power of C++. I love the power and the flexibility of this language."
+    },
+    {
+        "github_username": "arishuynhvan",
+        "name": "Ann TV Huynh",
+        "link_text": null,
+        "link_url": null,
+        "avatar_url": null,
+        "bio": "I worked with C++ during my first job as a developer 4 years ago and continued to use it in my scientific thesis. I love the power of this language and want to further refine my mastery over its various uses."
     }
+    
```

Comment:
No need for this empty line, please remove it

Reply:
Thanks for reviewing my PR. I've made the necessary changes.